docs/neon_os/using_neon_os.md

```diff
@@ -15,4 +15,20 @@ network.
 After network setup is complete, there will be some skills loading in the background;
 when those are loaded, the spinner in the top right corner of the screen will 
 animate and the homescreen will be displayed. At this point, Neon is set up and
-ready to go!
\ No newline at end of file
+ready to go!
+
+## Resetting
+To reset Neon to original settings, there is a menu option under `Settings` ->
+`Factory Settings` -> `Reset Device`. This removes any user settings, restores a
```

Comment:
'Reset Device' --> 'Factory Reset'   (the button label is factory reset, not reset device)